src/sentry/lang/native/utils.py

```diff
@@ -12,7 +12,9 @@
 WINDOWS_PATH_RE = re.compile(r"^([a-z]:\\|\\\\)", re.IGNORECASE)
 
 # Event platforms that could contain native stacktraces
-NATIVE_PLATFORMS = ("cocoa", "native")
+# il2cpp events have the "csharp" platform, and we do need to run those
+# through symbolicator to correctly symbolicate il2cpp stack frames.
+NATIVE_PLATFORMS = ("cocoa", "native", "csharp")
```

Comment:
just to confirm...we shouldn't update this in the [UI](https://github.com/getsentry/sentry/blob/ba3ecb9c6aba5b8f027817208f1327816d62b67f/static/app/utils/displayReprocessEventAction.tsx#L11) too right?

Reply:
I don’t think so.